karapace/protobuf/enum_element.py

```diff
@@ -9,7 +9,7 @@
 class EnumElement(TypeElement):
     constants: list = []
```

Comment:
Another comment, there are a few places with this pattern in the constructors:
```python
        if options:
            self.options = options
```
I assume this is trying to use the value in the class definition as the default, and only overwrite it if the user provided a useful value. Probably the pattern you're looking for is:

```python
self.options = options or default
self.options = options or list()
```
The `or` expression returns `options` if it is considered "truthy", in this case would be a non-empty list. Otherwise it evaluates and returns the second value `list()`.
```python
>>> from typing import Optional
>>> class A:
...   def __init__(self, options: Optional[list] = None) -> None:
...     self.options = options or list()
... 
>>> A().options
[]
>>> A([1]).options
[1]
```

Reply:
fixed